packages/hoc-disabled-tooltip/src/disabledTooltip.story.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { withInfo } from "@storybook/addon-info";
+
+import disabledTooltip from ".";
+
+const EnhancedInput = disabledTooltip("input");
+const EnhancedButton = disabledTooltip("button");
+
+storiesOf("HOC disabledTooltip", "module")
+  .add(
+    "disabled html input",
+    withInfo()(() => <EnhancedInput disabled type="text" />)
+  )
+  .add(
+    "disabled html input with tooltip right aligned",
+    withInfo()(() => (
+      <EnhancedInput disabled type="text" tooltipAlign="right" />
+    ))
+  )
+  .add(
+    "disabled html button",
+    withInfo()(() => (
+      <EnhancedButton disabled>A disabled Button</EnhancedButton>
+    ))
+  )
+  .add(
+    "disabled false and button events (click, mouseover)",
+    withInfo()(() => (
+      <EnhancedButton
+        onClick={action("button clicked")}
+        onMouseOver={action("mouse over")}
+        onMouseLeave={action("mouse out")}
+      >
+        A Button with events
+      </EnhancedButton>
+    ))
+  )
+  .add(
+    "disabled html input with custom Tooltip text",
+    withInfo()(() => (
+      <EnhancedInput
+        type="checkbox"
+        disabled
+        tooltipText="A custom tooltip text"
+      />
+    ))
+  );
```

Comment:
I think it'd be nice to have an example passing a React component to the hoc instead of only passing the `type` as string. I get that the output is the same, but for documentation sake (:

Reply:
The usage on the Readme has such an example now, but I can add one to the stories as well...